Automattic/wp-calypso
# Continue with Google: Fix failing e2e test (missing "Continue" button click)

Resolves https://github.com/Automattic/wp-calypso/issues/89243.

## Proposed Changes

* count with the new "Terms and conditions" step in the Google authorization flow and click on the "Continue" button

TODO:

- [ ] once the PR is merged, the related e2e test needs to be unmuted 

## Testing Instructions

1. Check out the PR branch.
2. Run Calypso locally by following the **Node.js server at port 443** setup steps: PCYsg-5YE-p2.
3. Navigate to `/tests/e2e`.
4. Run `yarn jest /test/e2e/specs/onboarding/signup__woo-google.ts`.
5. Observe the test running. It should not get stuck at the "Terms and conditions" step and pass correctly. 

## Pre-merge Checklist
